packages/dx-react-chart-demos/src/demo-sources/zoom-pan/scatter.partial.jsxt

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  ArgumentScale,
+  ValueScale,
+  EventTracker,
+  HoverState,
+  Animation,
+} from '@devexpress/dx-react-chart';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  ScatterSeries,
+  Tooltip,
+  ZoomAndPan,
+  Title,
+  Legend,
+} from '@devexpress/dx-react-chart-<%&themeName%>';
+<%&imports%>
+import { format } from 'd3-format';
+
+import { lifeBirth } from '../../../demo-data/life-birth';
+
+const getArgField = name => `birthRate_${name}`;
+const getValField = name => `lifeExp_${name}`;
+
+const data = lifeBirth.map(({
+  birthRate, lifeExp, year, country,
+}) => ({
+  [getArgField(year)]: birthRate,
+  [getValField(year)]: lifeExp,
+  year,
+  country,
+}));
+
+const adjustDomain = ([start, end]) => [Math.floor(start), Math.ceil(end)];
+
+const formatTooltipValue = format('.2f');
+
+<%&body%>
+
+const TooltipContent = ({ targetItem }) => {
+  const item = data[targetItem.point];
+  const arg = formatTooltipValue(item[getArgField(targetItem.series)]);
+  const val = formatTooltipValue(item[getValField(targetItem.series)]);
+  return (
+    <div>
+      <TooltipHeader>{`${item.country} ${item.year}`}</TooltipHeader>
+      <TooltipSplitter />
+      <table>
+        <tr>
+          <td><TooltipCell>Life Expectancy</TooltipCell></td>
+          <td><TooltipCell>{arg}</TooltipCell></td>
+        </tr>
+        <tr>
+          <td><TooltipCell>Birth Rate</TooltipCell></td>
+          <td><TooltipCell>{val}</TooltipCell></td>
+        </tr>
+      </table>
```

Comment:
Maybe make tooltip design like in the [devextreme charts demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/Overview/jQuery/Light/): with bold text in the header and add opacity to the splitter, `Life Expectancy` and `Birth Rate` phrases?

Reply:
Here is how it looks now.
![pr2058_bs4](https://user-images.githubusercontent.com/2501124/58704178-cf312580-83b3-11e9-989b-166211d411f4.png)
![pr2058_mui](https://user-images.githubusercontent.com/2501124/58704184-d22c1600-83b3-11e9-8802-fb378d82e4c5.png)

Headers visually are not so bold as those in *devextreme demo* but they have `font-weight: 500` like in *devextreme demo*. With `600` they would look rather ugly.

----
The amount of non-similarity between *bs4* and *mui* markup is TERRIBLE though.